src/workloads/sharding/WouldChangeOwningShardBatchWrite.yml

```diff
@@ -93,4 +93,5 @@ AutoRun:
           - shard-lite-80-feature-flags
       branch_name:
         $neq:
```

Comment:
Instead of listing versions, you can use $gte: v6.0

Reply:
Done